src/app/app.component.ts

```diff
@@ -1,9 +1,19 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from './shared/services/auth.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
+  constructor(private authService: AuthService) { }
+  ngOnInit(): void {
+    const token = this.authService.getToken();
```

Comment:
app.component should not know about the token. this logic must be under auth.service.